tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -4,34 +4,41 @@ const { series } = require('async');
 const { scalityS3Client, awsS3Client } = require('../../../s3SDK');
 const LifecycleUtility = require('../../LifecycleUtility');
 
-const srcBucket = `transition-bucket-${uuid()}`;
-const keyPrefix = uuid();
-
-const cloudServer = new LifecycleUtility(scalityS3Client)
-    .setBucket(srcBucket)
-    .setKeyPrefix(keyPrefix);
-
-const cloud = new LifecycleUtility(awsS3Client).setKeyPrefix(keyPrefix);
-
-function compareTransitionedData(cb) {
+function compareTransitionedData(sourceClient, destinationClient, version, cb) {
     return series([
-        next => cloudServer.getObject(next),
-        next => cloudServer.putBucketLifecycleConfiguration(new Date(), next),
-        next => cloudServer.waitUntilTransitioned(next),
-        next => cloud.getObjectDataFromLocation(next),
-        next => cloudServer.getObject(next),
+        next => sourceClient.getObject(next),
+        next => sourceClient.putBucketLifecycleConfiguration(new Date(), next),
+        next => sourceClient.waitUntilTransitioned(version, next),
+        next => destinationClient.getObjectDataFromLocation(next),
+        next => sourceClient.getObject(next),
     ], (err, data) => {
         if (err) {
             return cb(err);
         }
         // check data stored on the target cloud
-        cloudServer._compareObjectBody(data[0].Body, data[3]);
+        sourceClient._compareObjectBody(data[0].Body, data[3]);
         // check that object is still readable from Zenko
-        cloudServer._compareObjectBody(data[0].Body, data[4].Body);
+        sourceClient._compareObjectBody(data[0].Body, data[4].Body);
         return cb();
     });
 }
 
+function compareTransitionedColdData(sourceClient, version, cb) {
+    return series([
+        next => sourceClient.getObject(next),
+        next => sourceClient.putBucketLifecycleConfiguration(new Date(), next),
+        next => sourceClient.waitUntilTransitioned(version, next),
+    ], cb);
+}
+
+function checkTransition(destination, sourceClient, destinationClient, version, cb) {
+    if(destination.isCold) {
```

Comment:
```suggestion
    if (destination.isCold) {
```
Do we not have eslint enabled here? I feel like this is one of the rules.